src/Serilog/Serilog.csproj

```diff
@@ -0,0 +1,74 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Simple .NET logging with fully-structured events</Description>
+    <VersionPrefix>2.4.1</VersionPrefix>
+    <Authors>Serilog Contributors</Authors>
+    <TargetFrameworks>net45;net46;netstandard1.0;netstandard1.3</TargetFrameworks>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AssemblyName>Serilog</AssemblyName>
+    <AssemblyOriginatorKeyFile>../../assets/Serilog.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
```

Comment:
Actually I'm not sure now - I don't know what to do with this.

<img width="860" alt="screen shot 2017-03-08 at 16 57 31" src="https://cloud.githubusercontent.com/assets/570470/23693467/5b9c968c-0420-11e7-953f-d2ad672525c3.png">